src/scraper.rs

```diff
@@ -140,6 +140,14 @@ fn fetch(
                 }
             };
 
+            let line = match lib::remove_labels(&line, &scraper.filtered_labels) {
```

Comment:
check before the match that `filtered_labels` is empty

Reply:
Followed design from above to be consistent between add and remove_labels. add_labels does not check if labels is empty